packages/frontend/src/components/DashboardFilter/FilterConfiguration/index.tsx

```diff
@@ -248,6 +261,12 @@ const FilterConfiguration: FC<Props> = ({
         ],
     );
 
+    const showApplyButton = !isFilterConfigurationApplyButtonEnabled(
```

Comment:
This variable name should be different. 
I would go for : 
```suggestion
    const isApplyButtonDisabled = !isFilterConfigurationApplyButtonEnabled(
```

Reply:
Oops, I don't know how I got that backwards 😄

Reply:
Done